assets/animations.js

```diff
@@ -13,10 +13,18 @@ function onIntersection(elements, observer) {
   });
 }
 
-function initializeScrollAnimationTrigger(rootEl = document) {
+function initializeScrollAnimationTrigger(event, rootEl = document) {
```

Comment:
It seems like the only point of passing in `event` here is the check on line 22.

Would it be more self-explanatory to have a parameter like `isEditorEvent` instead which is a boolean?